sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -1002,7 +1002,7 @@ def default_label(self):
     return label_from_callable(self._fn)
 
 
-def ptransform_fn(fn):
+def ptransform_fn(fn) -> Callable[..., _PTransformFnPTransform]:
```

Comment:
We currently use annotations with explicit `# type:` comment in annotations. We added  them back when still supporting Python 2, and planned a rewrite into native Py3 type hints, but looks like that didn't land yet.
For consistency reasons, could you please you try the old style annotation and see if that works for your purposes, @SandyChapman ?

Reply:
Sure thing. I'll edit, test, and push the change shortly.

Reply:
@tvalentyn I've updated the PR.